STX.REST.RESTFulSense.Clients.Tests.Unit/Services/Foundations/LocalHttpClients/LocalHttpClientTests.Logic.Get.cs

```diff
@@ -40,23 +40,18 @@ public async Task ShouldGetAsync()
             // when
             LocalHttpClient actualLocalHttpClient =
                 await this.localHttpClientService.GetAsync(inputLocalHttpClient);
-
-
+            
             // then
             actualLocalHttpClient.Should().BeEquivalentTo(
                 expectedLocalHttpClient,
                 options => options.Excluding(x => x.HttpResponse.StreamContent));
 
             using MemoryStream actualContentStream = new MemoryStream();
             await actualLocalHttpClient.HttpResponse.StreamContent.CopyToAsync(actualContentStream);
-
```

Comment:
I would not remove this line. What do you think @glhays

Reply:
See discord comment.

Reply:
Agree with that,
```csharp
        [Fact]
        private async Task ShouldSendLocalClientRequestWhenGetAsyncIsCalledAsync()
        {
            // given
            dynamic randomProperties = CreateRandomProperties();

            LocalHttpClient inputLocalHttpClient =
                CreateHttpClient(randomProperties);

            LocalHttpClient expectedLocalHttpClient =
                CreateHttpClientResponse(
                    inputLocalHttpClient,
                    randomProperties);

            HttpRequestMessage inputHttpRequestMessage =
                CreateHttpRequestMessage(randomProperties);

            HttpResponseMessage expectedHttpResponseMessage =
                CreateHttpResponseMessage(randomProperties);

            this.httpClientBroker.Setup(broker =>
                broker.SendRequestAsync(
                    It.IsAny<HttpRequestMessage>(), default))
                        .ReturnsAsync(expectedHttpResponseMessage);

            // when
            LocalHttpClient actualLocalHttpClient =
                await this.localHttpClientService.GetAsync(
                    inputLocalHttpClient);

            // then
            actualLocalHttpClient.Should().BeEquivalentTo(
                expectedLocalHttpClient,
                options => options.Excluding(
                    x => x.HttpResponse.StreamContent));

            using MemoryStream actualContentStream =
                new MemoryStream();

            await actualLocalHttpClient.HttpResponse.StreamContent
                .CopyToAsync(actualContentStream);

            using MemoryStream expectedContentStream =
                new MemoryStream();
            
            await expectedLocalHttpClient.HttpResponse.StreamContent
                .CopyToAsync(expectedContentStream);

            actualContentStream.ToArray().Should().BeEquivalentTo(
                expectedContentStream.ToArray());

            this.httpClientBroker.Verify(broker =>
                broker.SendRequestAsync(
                    It.IsAny<HttpRequestMessage>(), default),
                        Times.Once);

            this.httpClientBroker.VerifyNoOtherCalls();
        }

```